pokerpoll/pokerpoll.py

```diff
@@ -1,56 +1,79 @@
 import discord
 from discord.ext import commands
 
+
```

Comment:
in future split up commits into logical changes (in this case, whitespace changes and your time additions)